README.rst

```diff
@@ -104,20 +117,20 @@ ACME working area in github: https://github.com/ietf-wg-acme/acme
 .. _Freenode: https://webchat.freenode.net?channels=%23letsencrypt
 
 .. |build-status| image:: https://travis-ci.org/certbot/certbot.svg?branch=master
-   :target: https://travis-ci.org/certbot/certbot
-   :alt: Travis CI status
+   :target: https://travis-ci.org/certbot/certbot
```

Comment:
It looks like there's still a few wrong spaces here that need to be fixed

Reply:
I'll take a look.